epsagon_kubernetes_subscription.sh

```diff
@@ -0,0 +1,105 @@
+#! /bin/bash
+
+## Script to attach Epsagon Role to kubernetes
+
+function usage {
+    echo "Usage: epsagon_kubernetes_subscription.sh EPSAGON_TOKEN"
+}
+
+ROLE_URL=https://raw.githubusercontent.com/epsagon/epsagon-k8s-role/master/epsagon-role.yaml
```

Comment:
A question regarding the role - 
Why are the service account and namespace `epsagon-monitoring` but the clusterRole and clusterRoleBinding are `epsagon-prometheus`? (I am using those privileges with REST without prometheus)

Reply:
We can change it (in a different PR)